packages/redux-variants/variants.js

```diff
@@ -383,6 +412,24 @@ const filteredVariantsById = createSelector(
       filteredVariants = filteredVariants.filter(v => searchPredicate(v, query))
     }
 
+    // Indel and Snp filters.
+    filteredVariants = filteredVariants.filter(v => {
+      let splits = v.get('variant_id').split('-')
+      if (splits.length === 4) {
```

Comment:
It's safe to assume that variant IDs have 4 parts.

Reply:
Do we do validation somewhere before loading? My only concern with this is if for some reason this isn't true, the page will break (unless we have a root exception handler that swallows it).

Reply:
It's not validated anywhere, but it's defined as `chrom-pos-ref-alt` in our variant datasets.

https://github.com/macarthur-lab/hail-elasticsearch-pipelines/blob/4322dc60a22eb42d1ac35a90ad4df158c0421c61/hail_scripts/v02/utils/computed_fields/variant_id.py#L46-L59